diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6656871..0b4fd14 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,18 +16,18 @@ if(CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
     configure_file(${CMAKE_SOURCE_DIR}/README-tmpl.md                        ${CMAKE_SOURCE_DIR}/README.md @ONLY)
     configure_file(${CMAKE_SOURCE_DIR}/bindings/python/setup-tmpl.py         ${CMAKE_SOURCE_DIR}/bindings/python/setup.py @ONLY)
     configure_file(${CMAKE_SOURCE_DIR}/bindings/javascript/package-tmpl.json ${CMAKE_SOURCE_DIR}/bindings/javascript/package.json @ONLY)
-    configure_file(${CMAKE_SOURCE_DIR}/bindings/ios/Makefile-tmpl            ${CMAKE_SOURCE_DIR}/bindings/ios/Makefile @ONLY)
+#    configure_file(${CMAKE_SOURCE_DIR}/bindings/ios/Makefile-tmpl            ${CMAKE_SOURCE_DIR}/bindings/ios/Makefile @ONLY)
 else()
     set(GGWAVE_STANDALONE OFF)
 endif()
 
 if (EMSCRIPTEN)
-    set(GGWAVE_SUPPORT_SDL2_DEFAULT ON)
+    set(GGWAVE_SUPPORT_SDL2_DEFAULT OFF)
     set(BUILD_SHARED_LIBS_DEFAULT OFF)
 
     option(GGWAVE_WASM_SINGLE_FILE "ggwave: embed WASM inside the generated ggwave.js" ON)
 else()
-    set(GGWAVE_SUPPORT_SDL2_DEFAULT ON)
+    set(GGWAVE_SUPPORT_SDL2_DEFAULT OFF)
     if (WIN32)
         set(BUILD_SHARED_LIBS_DEFAULT OFF)
     else()
diff --git a/examples/CMakeLists.txt b/examples/CMakeLists.txt
index 6c6e3b9..2a550fd 100644
--- a/examples/CMakeLists.txt
+++ b/examples/CMakeLists.txt
@@ -43,7 +43,7 @@ endif()
 
 # third-party
 
-add_subdirectory(third-party)
+#add_subdirectory(third-party)
 
 # helper libraries
 
